src/hotspot/share/opto/escape.cpp

```diff
@@ -3739,6 +3773,34 @@ void ConnectionGraph::dump(GrowableArray<PointsToNode*>& ptnodes_worklist) {
   }
 }
 
+void ConnectionGraph::print_statistics() {
+  tty->print("No escape = %d, Arg escape = %d, Global escape = %d", Atomic::load(&_no_escape_counter), Atomic::load(&_arg_escape_counter), Atomic::load(&_global_escape_counter));
+  tty->print_cr(" (EA executed in %7.2f seconds)", Atomic::load(&_time_elapsed) * 0.001);
+}
+
+void ConnectionGraph::escape_state_statistics(GrowableArray<JavaObjectNode*>& java_objects_worklist) {
+  _compile->_local_no_escape_ctr = 0;
+  _compile->_local_arg_escape_ctr = 0;
+  _compile->_local_global_escape_ctr = 0;
+  for (int next = 0; next < java_objects_worklist.length(); ++next) {
+    JavaObjectNode* ptn = java_objects_worklist.at(next);
+    if (ptn->ideal_node()->is_Allocate()) {
+      if(ptn->escape_state() == PointsToNode::NoEscape) {
+        _compile->_local_no_escape_ctr++;
+      }
+      else if (ptn->escape_state() == PointsToNode::ArgEscape) {
+        _compile->_local_arg_escape_ctr++;
+      }
+      else if (ptn->escape_state() == PointsToNode::GlobalEscape) {
```

Comment:
"else if" style is not consistent with others.

Reply:
NIT: Move "else if" to line 3793.